fbw-a380x/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -781,18 +782,16 @@ class AltimeterIndicator extends DisplayComponent<AltimeterIndicatorProps> {
   private handleBlink() {
     if (this.mode.get() === 'STD') {
       if (
-        this.flightPhase > 3 &&
-        this.transLvlAr.isNormalOperation() &&
-        100 * this.transLvlAr.value > this.props.altitude.get()
+        this.fmgcFlightPhase > FmgcFlightPhase.Cruise &&
+        100 * (this.transLvlAr.isNormalOperation() ? this.transLvlAr.value : 25) > this.props.altitude.get()
       ) {
         this.stdGroup.instance.classList.add('BlinkInfinite');
       } else {
         this.stdGroup.instance.classList.remove('BlinkInfinite');
       }
     } else if (
-      this.flightPhase <= 3 &&
-      this.transAltAr.isNormalOperation() &&
-      this.transAltAr.value < this.props.altitude.get()
+      this.fmgcFlightPhase <= FmgcFlightPhase.Cruise &&
+      (this.transAltAr.isNormalOperation() ? this.transAltAr.value : 2_500) < this.props.altitude.get()
```

Comment:
No default for trans alt. It should not blink if the trans level is NCD or FW.

Reply:
same here as before, will use old logic